### DISCUSSION TITLE
"remove all upwards" and "remove all downwards"

### DISCUSSION THREAD

**Discussion by GK-774** [jcorporation/myMPD#610] [General]
In the queue "remove all upwards" and "remove all downwards" always includes the selected song as well. Should it not remove downwards all songs below the selected and upwards those above?
(myMPD 8.1.6)

**Comment by jcorporation** [comment]
Right, this would be the correct behavior. It is already fixed in the devel branch (upcoming 8.2.0 release).

**Comment by GK-774** [comment]
Ok, thanks.
